### PR TITLE
[KBFS-2756] Show Folder in Finder with Popup

### PR DESCRIPTION
**Pull request by jzila**
screen shot 2018-03-14 at 10 45 20 am

### CODE REVIEW

**Comment by jzila** [comment]
Ready for review

File: shared/fs/header/breadcrumb-popup.js [hunk 1]
```diff
@@ -33,14 +34,13 @@ const BreadcrumbPopupMenu = ({routeProps}: PopupMenuProps) => {
...
 }
 
 const stylePopup = {
-  position: 'absolute',
-  left: -28,
-  top: 24,
-  width: 196,
+  width: isMobile ? '100%' : 196,
+  marginTop: isMobile ? undefined : 12,
+  marginLeft: isMobile ? undefined : -12,
```
Comment (songgao) [inline shared/fs/header/breadcrumb-popup.js]: Could use `platformStyles` here? Not critical though so feel free to delay it until your next PR.

Comment (chrisnojima) [reply shared/fs/header/breadcrumb-popup.js]: Platform styles isn't really required if your styles are compatible with being cross platform. Mainly used when that's not true

Comment (songgao) [reply shared/fs/header/breadcrumb-popup.js]: Oh I thought that was just a cleaner way to do things. Thanks for the clarification.

Comment (chrisnojima) [reply shared/fs/header/breadcrumb-popup.js]: I think it is kinda cleaner and it'll match more code but it's not strictly required. In other places it is

**Pull request merged**